src/GlassCat/utilities.jl

```diff
@@ -318,3 +318,28 @@ function plot_indices(glass::AbstractGlass; polyfit::Bool = false, fiterror::Boo
 
     gui(p)
 end
+
+
+""" Draw a scatter plot of index vs dispersion (the derivative of index with respect to wavelength). Both index and dispersion are computed at wavelength λ. If showprefixglasses is true then glasses with names like F_BAK7 will be displayed. Otherwise glasses that have a leading letter prefix followed by an underscore, such as F_, will not be displayed."""
+function drawglassmap(glasscatalog::Module; λ = 550nm, glassfontsize = 3, showprefixglasses = false)
+    wavelength = Float64(ustrip(uconvert(u"μm", λ)))
+    indices = Vector{Float64}(undef,0)
+    dispersions = Vector{Float64}(undef,0)
+    glassnames = Vector{String}(undef,0)
+
+    for name in names(glasscatalog)
+        glass = eval(:($glasscatalog.$name))
+        glassstring = String(name)
+        hasprefix = occursin("_",glassstring)
```

Comment:
There are some glasses like NBFD15_W and FDS90_SG in HOYA. Should these count as having prefixes? I'm not familiar enough with naming conventions to know.

Reply:
Not sure what a suffix like that means. I believe prefixes indicate different versions of the same glass having nearly identical optical properties but slightly different physical properties, e.g., lead free vs  leaded, etc. If you don't filter these out then you end up with many glasses mapping to the same point on the glass map.

Reply:
It’s possible that invalid characters get replaced with underscores when the AGF files are parsed